src/Evolution/Executables/GeneralizedHarmonic/EvolveGeneralizedHarmonicSingleBlackHole.hpp

```diff
@@ -110,11 +151,21 @@ struct EvolutionMetavars
 
   using typename gh_base::step_actions;
 
+  // Add ElementInitInterpPoints to initialization_actions.
+  using helper_init =
+      tmpl::push_back<tmpl::pop_back<initialization_actions>,
+                      intrp::Actions::ElementInitInterpPoints<
+                          intrp::Tags::InterpPointInfo<EvolutionMetavars>>,
+                      tmpl::back<initialization_actions>>;
+
+  // the dg element array needs to be re-declared to capture the new type
+  // aliases for the action lists.
```

Comment:
Is this comment correct? It wasn't there before and isn't related to the change if I understand correctly. Maybe a remnant of a rebase?